src/platform/forms-system/src/js/web-component-patterns/fullNamePattern.js

```diff
@@ -140,6 +148,13 @@ const fullNameSchema = commonDefinitions.fullName;
  */
 const fullNameNoSuffixSchema = commonDefinitions.fullNameNoSuffix;
 
+/**
+ * @returns `commonDefinitions.fullNameNoSuffix` minus middle name
+ */
+const noMiddleNoSuffixSchema = commonDefinitions.fullNameNoSuffix;
```

Comment:
hm, this works. or what do you think of just adding firstAndLastNameUI, firstAndLastNameSchema, add not worrying about passing an omit property to fullNameSchema?

Reply:
Refactors done in new feature-branch.